bigquery/deploy/README.md

```diff
@@ -53,9 +53,10 @@
     ```
 
 * Run indexer on GKE
-  * If you are using the default [platinum_genomes dataset](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery/config/platinum_genomes),
-don't forget to [copy to your project and set project IDs in facet_fields.csv](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#quickstart).
-  * Build, tag, and upload the docker image to GCR:
+  * Make sure the config in `bigquery/dataset_config/MY_DATASET` is filled out.
```

Comment:
I prefer "config files", can you change it back? "config" isn't standard terminology.

Reply:
Done